ocpmodels/common/logger.py

```diff
@@ -52,15 +52,14 @@ def __init__(self, config):
             name=self.config["cmd"]["identifier"],
             dir=self.config["cmd"]["logs_dir"],
             project=self.config.get("logger_project", None),
-            reinit=True,
```

Comment:
This was just unnecessary correct? No issues caused as a result of it...

Reply:
Yes, this was before we made distributed the default